hw/ip/otbn/doc/dv/index.md

```diff
@@ -325,6 +336,145 @@ No special coverage points for this instruction.
 
 As for `LOOP`, but without the count of `'1` (not achievable with an immediate).
 
+#### BN.ADD
+
+- Extremal values of shift for both directions where the shifted value is nonzero
+- A nonzero right shift with a value in `wrs2` whose top bit is set
+
+#### BN.ADDC
+
+- Extremal values of shift for both directions where the shifted value is nonzero
+- A nonzero right shift with a value in `wrs2` whose top bit is set
+- Execute with both values of the carry flag for both flag groups (to make sure things are wired through properly)
+
+#### BN.ADDI
+
+As for `BN.ADD`.
+
+#### BN.ADDM
+
+- Execute with the two extreme values of `MOD` (zero and all ones)
+- Perform a subtraction (because the sum is at least `MOD`) when `MOD` is nonzero.
+- Don't perform a subtraction (because the sum is less than `MOD`) when `MOD` is nonzero.
+- Perform a subtraction where the sum is at least twice a nonzero value of `MOD`.
+- A calculation where the sum exactly equals a nonzero `MOD`
+
+#### BN.MULQACC
+
+- Cross `wrs1_qwsel` with `wrs2_qwsel` to make sure they are applied to the right inputs
+- See the accumulator overflow
+
+#### BN.MULQACC.WO
+
+As for `BN.MULQACC`, plus the generic flag group cover points.
+
+#### BN.MULQACC.SO
+
+As for `BN.MULQACC` plus the following:
+
+- See bits being cleared in the destination register as part of shifting out the result (makes sure that we're masking things properly when assembling the new value)
```

Comment:
the `.SO` variant doesn't clear any bits in the destination register, it just writes to one half or the other leaving the other bits untouched though it will clear bits in the accumulator.